### PR TITLE
Add FEN (event ports) Support to file monitor

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

**Comment by Toasterson** [comment]
Required by https://github.com/OpenIndiana/oi-userland/pull/16642

File: components/library/gamin/pkg5 [hunk 1]
```diff
@@ -1,13 +1,10 @@
 {
     "dependencies": [
-        "SUNWcs",
         "library/glib2",
-        "runtime/python-39",
-        "shell/ksh93",
```
Comment (AndWac) [inline components/library/gamin/pkg5:6]: You'll need to update pkg5 again.

File: components/library/gamin/Makefile [hunk 2]
```diff
@@ -46,12 +46,13 @@ COMPONENT_POST_INSTALL_ACTION.64 += \
 	$(MV) $(PROTO_DIR)$(PYTHON_LIB)/_gamin.so $(PROTO_DIR)$(PYTHON_LIB)/_gamin$$EXT_SUFFIX ;
 
 CONFIGURE_OPTIONS += --sysconfdir=/etc
-CONFIGURE_OPTIONS += --libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
 CONFIGURE_OPTIONS += --disable-static
 CONFIGURE_OPTIONS.32 += --with-python=no
 CONFIGURE_OPTIONS.64 += --with-python=/usr
+CONFIGURE_OPTIONS += DAEMON_LIBS="-lglib-2.0 -lgobject-2.0 -lgio-2.0"
+
+REQUIRED_PACKAGES += runtime/python-39
```
Comment (AndWac) [inline components/library/gamin/Makefile:55]: PYTHON_REQUIRED_PACKAGES += runtime/python
would be better

**Comment by Toasterson** [comment]
Both fixed.

File: components/library/gamin/Makefile [hunk 3]
```diff
@@ -46,12 +46,13 @@ COMPONENT_POST_INSTALL_ACTION.64 += \
 	$(MV) $(PROTO_DIR)$(PYTHON_LIB)/_gamin.so $(PROTO_DIR)$(PYTHON_LIB)/_gamin$$EXT_SUFFIX ;
 
 CONFIGURE_OPTIONS += --sysconfdir=/etc
-CONFIGURE_OPTIONS += --libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
 CONFIGURE_OPTIONS += --disable-static
 CONFIGURE_OPTIONS.32 += --with-python=no
 CONFIGURE_OPTIONS.64 += --with-python=/usr
+CONFIGURE_OPTIONS += DAEMON_LIBS="-lglib-2.0 -lgobject-2.0 -lgio-2.0"
+
+REQUIRED_PACKAGES += runtime/python
```
Comment (AndWac) [inline components/library/gamin/Makefile:55]: PYTHON_REQUIRED_PACKAGES!

**Pull request merged**